### ISSUE TITLE
The accuracy of validation set is 97%, and only 74% of the test results are obtained with validation set data. Why?

### ISSUE THREAD

**Issue by KouJiaWei** [Sierkinhane/crnn_chinese_characters_rec#98]
My training data is 30 X ? ，The height is fixed and the width is uncertain. Training data 10000, validation set data 2000, the accuracy of validation set in training process is 95%-99%, but when I test with validation set data, the accuracy is only 74%. What can I do to solve it?

**Comment by Sierkinhane** [comment]
Overfitting. You should do data augmentation or add regularization or cut down the network or add dropout.
